.github/workflows/build-unstable.yml

```diff
@@ -59,20 +55,46 @@ jobs:
         -   name: Setup Environment
             run: make setup
 
-        -   name: Build and Test
-            run: make test
+        -   name: Build
+            run: make build
+
+        -   name: TypeScript Tests
+            run: make test-ts
+
+        -   name: Python Tests
+            run: make test-py
```

Comment:
we should also run `test-integration`, right?

Reply:
Yes just fixed a merge conflict there, so this is how I intend it to look on merge.

So I just need a +1 here then I can do the `make example` changes discussed earlier.